code/modules/reagents/dispenser/dispenser_presets.dm

```diff
@@ -118,6 +118,23 @@
 			/obj/item/weapon/reagent_containers/chem_disp_cartridge/mead
 		)
 
+/obj/machinery/chemical_dispenser/coffeemaster
+	name = "Coffee Master 3000"

```

Comment:
Do we really need another coffe chem dispenser when there is one already on the code, with sprites?

Reply:
To the best of my knowledge, it sits in the bridge. The way I see it, it's a unique thing for command staff with its own mini chemistry system (`black_coffee`). It dispenses only black coffee.

If I put it in the bar it's going to take away from the one in the bridge.
Also, the coffeemaster dispenses not just coffee but also milk, soy, sugar, and espresso. Does it make sense to add them to the coffee maker in the bridge too?

Reply:
The coffee machine is legacy code from an old spriter from last year who tried to code a functionally equivalent coffee system as to the one you are coding now, but left the server before it could be finished. If you are introducing a new system, incorporate or remove the old one.

Reply:
Done.